docs/getting-started/integrations-cdn/vuejs-v2.md

```diff
@@ -0,0 +1,147 @@
+</info-box>
+
+Vue.js is a versatile framework for building web user interfaces. CKEditor&nbsp;5 provides the official Vue component you can use in your application. This guide assumes that you already have a Vue 2 project.
+
+{@snippet getting-started/use-builder}
+
+## Quick start
+
+First, install the CKEditor&nbsp;5 WYSIWYG editor component for Vue 2:
+
+```bash
+npm install @ckeditor/ckeditor5-vue2
+```
+
+Then, include the CKEditor&nbsp;5 scripts and styles. All necessary scripts and links are in the HTML snippet below. You can copy and paste them into your `index.html`. Open-source and premium features are in separate files, so there are different tags for both types of plugins. Add tags for premium features only if you use them.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" href="/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		
+		<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+		<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.umd.js"></script>
+		<!-- Add if you use premium features -->
+        <link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5-premium-features/{@var ckeditor5-version}/ckeditor5-premium-features.css" />
+        <script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5-premium-features.umd.js"></script>
+		<!--  -->
+
+		<title>Vite App</title>
+	</head>
+	<body>
+		<div id="app"></div>
+		<script type="module" src="/src/main.js"></script>
+	</body>
+</html>
+
+```
+
+To create an editor instance, you must first import the editor and the component modules into the root file of your application (for example, `main.js` when generated by create-vue).
+
+```js
+import Vue from 'vue';
+import CKEditor from '@ckeditor/ckeditor5-vue2';
+import App from './App.vue';
+
+Vue.use( CKEditor );
+
+new Vue( { render: ( h ) => h( App ) } ).$mount( '#app' );
+```
+
+Use the `<ckeditor>` component inside the template tag. The below example shows how to use the component with open-source plugins.
+
+```html
+<template>
+	<div id="app">
+		<ckeditor :editor="editor" v-model="editorData" :config="editorConfig"></ckeditor>
+	</div>
+</template>
+
+<script>
+const {
+	ClassicEditor,
+	Essentials,
+	Bold,
+	Italic,
+	Paragraph
+} = CKEDITOR;
+
+export default {
+	name: 'app',
+	data() {
+		return {
+			editor: ClassicEditor,
+			editorData: '<p>Hello from CKEditor 5 in Vue 2!</p>',
+			editorConfig: {
+				licenseKey: 'GPL',
+				plugins: [ Bold, Essentials, Italic, Paragraph ],
+				toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ]
+			}
+		};
+	}
+};
+</script>
+```
+
+The following example shows how to use the component with premium plugins. To make it work, you need to provide a proper license key.
```

Comment:
I deliberately divided the snippet into two. The snippet with premium plugins caused errors (globals not available). The open-source works after a simple copy-paste.

Reply:
The guide doesn't appear so poor, so that's another plus, I guess.